services/editor/Dockerfile

```diff
@@ -15,6 +18,7 @@ RUN yarn
 
 COPY . /src/
 RUN yarn test
+RUN yarn run compile-less
```

Comment:
I thought about it, the problem is that `compile-less` is a bash script, so if you run `build` in windows it will fail